### ISSUE TITLE
systemctl stop does not stop everything

### ISSUE THREAD

**Issue by DrDaveD** [cvmfs-contrib/cvmfs-stratum1-replicator#6]
The stop-cvmfs-snapshots command stops everything, but systemctl stop doesn't.  Perhaps there could be a separate service called cvmfs-stratum1-replicator which invokes the other services via stop-cvmfs-snapshots.  Maybe that would also help with #5.

**Comment by bbockelm** [comment]
What are you running for `systemctl stop`?  Do you mean `systemctl stop cvmfs-snapshot`?  The systemd config looks correct to me.

Would be useful to know what systemd says about the individual service and timers after that command is run.

**Comment by DrDaveD** [comment]
Yes, I meant systemctl stop cvmfs-snapshot.  See [stop-cvmfs-snapshots](https://github.com/cvmfs-contrib/cvmfs-stratum1-replicator/blob/master/stop-cvmfs-snapshots) for the steps that are needed to stop everything.

**Comment by bbockelm** [comment]
The intent is certainly to have it do everything except `rm -f /var/spool/cvmfs/$d/is_updating.lock` (but that can be added as a post-script).

Is this easily reproducible?  What's the best place to debug Stratum-1 issues these days?

**Comment by DrDaveD** [comment]
Yes, it is reproducible.  You can test on hcc-cvmfs-itb.
